lib/pure/strutils.nim

```diff
@@ -1114,7 +1114,7 @@ func parseFloat*(s: string): float {.rtl, extern: "nsuParseFloat".} =
   if L != s.len or L == 0:
     raise newException(ValueError, "invalid float: " & s)
 
-func parseBinInt*(s: string): int {.rtl, extern: "nsuParseBinInt".} =
+func parseBinInt*(s: string): int {.deprecated: "use parseBin", rtl, extern: "nsuParseBinInt".} =
```

Comment:
```
deprecated: "use `strutils.parseBin`"
```
to avoid confusion with parseutils.parseBin ?
ditto with other deprecations below